.agents/skills/git-worktree/scripts/worktree-manager.sh

```
@@ -0,0 +1,500 @@
+  echo ""
+  echo -e "Remove $found worktree(s)? (y/n)"
+  read -r response
+
+  if [[ "$response" != "y" ]]; then
+    echo -e "${YELLOW}Cleanup cancelled${NC}"
+    return
+  fi
+
+  echo -e "${BLUE}Cleaning up worktrees...${NC}"
+  for worktree_path in "${to_remove[@]}"; do
+    local worktree_name=$(basename "$worktree_path")
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)